man/systemd.network.xml

```diff
@@ -1385,6 +1385,18 @@ IPv6Token=prefixstable:2002:da8:1::</programlisting></para>
             <literal>no</literal>.</para>
           </listitem>
         </varlistentry>
+        <varlistentry>
+          <term><varname>Group=</varname></term>
+          <listitem>
+            <para>Takes a whitespace separated list of nexthop IDs. Each ID must be in the range
+            1…4294967295. Optionally, each nexthop ID can take a weight after a comma
+            (<literal><replaceable>id</replaceable>,<replaceable>weight</replaceable></literal>). The
```

Comment:
hmm, a comma? usually we use commas as separators in lists, i.e. to separate multiple elements of the same type. But here it's used to separate multiple elements that are conceptually different.

Is this inherited from "iproute" syntax?

Otherwise, I think using a colon would be better, as that doesn#t convey the type equivalency of its lhs + rhs so much I think.